Models/JsonStat/SSBAgesInNorway.cs

```diff
@@ -0,0 +1,31 @@
+using System.Text.Json.Serialization;
+
+namespace Backend.Models.JsonStat
+{
+    // Remove comments when merge-ready
+    // Based on https://data.ssb.no/api/v0/dataset/1074.json?lang=no
+    public class SSBAgesInNorway
```

Comment:
Can we abstract even further?

Reply:
I believe you can make the `class` generic, and make the `Dimension` property use this generic type.